Dockerfile

```diff
@@ -1,8 +1,11 @@
 # Base image
-ARG LITELLM_BASE_IMAGE=python:3.9-slim
+ARG LITELLM_BASE_IMAGE=python:3.9
 
-# allow users to specify, else use python 3.9-slim
-FROM $LITELLM_BASE_IMAGE
+# Runtime image
+ARG LITELLM_RUNTIME_IMAGE=python:3.9-slim
```

Comment:
Come to think of it, can we get away with `alpine` as the default?

@krrishdholakia Foresee any dependencies that might not play well with ARM?

Reply:
yeh alpine or wolfi base https://edu.chainguard.dev/open-source/wolfi/wolfi-with-dockerfiles/